src/radical/pilot/utils/component.py

```diff
@@ -1429,8 +1429,12 @@ def publish(self, pubsub, msg):
         self.is_valid()
 
         if pubsub not in self._publishers:
-            raise RuntimeError("can't route '%s' notification: %s" % (pubsub,
+            self._log.warn("can't route '%s' notification: %s" % (pubsub,
```

Comment:
What is the reason that this is reduced from an error to a warning? I am not say it should stay as an error, I just to be sure.